static_test_driver/sensors.ino

```diff
@@ -2,7 +2,7 @@ float mean (const float *data, unsigned int size) {
   float result;
```

Comment:
Oh, one more thing I forgot: You shouldn't leave result uninitialized.

Reply:
Lol, yeah that would have been bad. Sorry about that.